Improve README.md

creativetags opened the pull request:
It took me quite a while to figure out how to get this running. I'm suggesting some additional info in the readme to help others.

creativetags committed:
Improve README.md

tcannonfodder reviewed (commented):
Thanks for submitting a PR! I agree, the documentation is sorely needing of improvements (we've been focusing our documentation efforts on `devise-passkeys`).

This is a great set of changes. I'm going to edit them slightly for some formatting & styling choices.

tcannonfodder committed:
Edit the README for clarity & styling

* List out the general requirements, not specifying any
	platform-specific & tool-specific installation instructions in order
	to reduce maintenance overhead
* Copyedting
